src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -266,9 +266,8 @@ export default {
 		preFetch: true,
 		result({ data }) {
 			this.isMonthlyGoodSubscriber = data?.my?.autoDeposit?.isSubscriber ?? false;
-			// TODO! Add this back in when service supports non-logged in users
-			// const modernSubscriptions = data?.mySubscriptions?.values ?? [];
-			// this.hasModernSub = modernSubscriptions.length !== 0;
+			const modernSubscriptions = data?.mySubscriptions?.values ?? [];
+			this.hasModernSub = modernSubscriptions.length !== 0;
```

Comment:
So no more failures for non-logged-in or non-existent users in the subscriptions service?

Reply:
Correct this was fixed a while ago, but the service now returns null and behaves the same as `my.account` -- See [PBJ-224](https://kiva.atlassian.net/browse/PBJ-224)